feature-configs/hypershift-ci/container-mount-namespace/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../deploy/container-mount-namespace/
```

Comment:
please check I think this use a MCP that is not supported

Reply:
yes, removing